Md4c switch

Waqar144 opened the pull request.

pbek commented on src/entities/note.cpp:
```diff
@@ -2138,9 +2155,7 @@ QString Note::textToMarkdownHtml(QString str, const QString& notesPath,
     // do some more code formatting
-    // the "pre" styles are for the full-width code block background color
     codeStyleSheet += QString(
-                    "pre { display: block; background-color: %1 } "
```
I don't think you want to remove that, do you?

Waqar144 replied:
Nah, leftover. I will fix that.

pbek commented on src/entities/note.cpp:
```diff
@@ -2138,7 +2135,6 @@ QString Note::textToMarkdownHtml(QString str, const QString& notesPath,
     QString codeBackgroundColor = darkModeColors ? QStringLiteral("#444444") : QStringLiteral("#f1f1f1");
 
     // do some more code formatting
-    // the "pre" styles are for the full-width code block background color
```
can you please leave this in too

Waqar144 replied:
ok :grin:

pbek commented:
This is strange:

![image](https://user-images.githubusercontent.com/1798101/72153452-4c8f3080-33ae-11ea-948e-a83608df6300.png)

    
```
    .boton2 
    {
        background: #898989;
        border:0px;
        color:White;
        border-radius: 3px;
        padding: 6px 20px;
        font-size: 14px;
    }
    .boton2:focus 
    {
        outline:0px;
    }
    .boton2:hover 
    {
        background: #6E6E6E;   
    }
```

pbek commented:
Damn, it's the same with hoedown. Another code block regression.

Waqar144 commented:
> Damn, it's the same with hoedown. Another code block regression, I updated the issue.

Yeah. Just fixed though.

It was an oversight on my part.
